app/controllers/v0/sign_in_controller.rb

```diff
@@ -232,12 +232,14 @@ def logout_get_redirect_url
     end
 
     def handle_pre_login_error(error, client_id)
-      if SignIn::Constants::ClientConfig::COOKIE_AUTH.include?(client_id)
+      if client_config(client_id).cookie_auth?
         error_code = error.try(:code) || SignIn::Constants::ErrorCode::INVALID_REQUEST
         redirect_to failed_auth_url({ auth: 'fail', code: error_code, request_id: request.request_id })
       else
         render json: { errors: error }, status: :bad_request
       end
+    rescue ActiveModel::ValidationError
```

Comment:
Ahhh, I see what you were trying to do here. I didn't realize that `client_config` itself could raise an error just by being invoked. Honestly I'd prob remove this `rescue` and just let the route do a hard failure if this happens. It should never happen (as the validation should have already occurred for `auth`, and shouldn't be possible in the `state_payload` for `callback`, so if it starts happening we actually probably do want it to hard fail and get a bunch of sentry notifications